packages/tsx-editor/src/utilities/isEditorSupported.ts

```diff
@@ -11,8 +10,6 @@ export function isEditorSupported(code: string, supportedPackages: IBasicPackage
     !!win &&
     // Required environment config available
     !!getMonacoConfig() &&
-    // Opt-in query param or session storage is set
-    getSetting('useEditor') === '1' &&
```

Comment:
Maybe we should keep a flag here allow to do:

```
useEditor = 0
```

to turn off the feature?

Reply:
Yeah I think that's a good idea.